gtsam/hybrid/tests/testGaussianMixture.cpp

```diff
@@ -133,7 +117,11 @@ TEST(GaussianMixture, GaussianMixtureModel2) {
   double mu0 = 1.0, mu1 = 3.0;
   double sigma0 = 8.0, sigma1 = 4.0;
 
-  auto hbn = GaussianMixtureModel(mu0, mu1, sigma0, sigma1);
+  HybridBayesNet hbn;
```

Comment:
Comment about `GaussianMixtureModel`. The function was self-documenting in a way.

Reply:
Ok